docs/examples/groups.md

```diff
@@ -0,0 +1,61 @@
+# Messages <Badge text="example"/>
+
+This example create a function ``xor_range`` that compute the xor in range ``[a, b]``.
+It is shown how to update the progress, as well as add middlewares to cancel and save the state.
+
+<CImage src="example_groups.png" caption="Result Example tab /groups" zoom="true"></CImage>
+
+## Script
+
+File name: ``groups.py``
+
+``` python
+import random
+
+import remoulade
+from remoulade import group
+from remoulade.api.main import app
+from remoulade.brokers.rabbitmq import RabbitmqBroker
+from remoulade.cancel import Cancel
+from remoulade.cancel.backends import StubBackend as CancelBackend
+from remoulade.middleware import CurrentMessage
+from remoulade.state import MessageState, backends
+
+broker = RabbitmqBroker()
+# allow to cancel a message
+broker.add_middleware(Cancel(backend=CancelBackend()))
+# allow to update the progress
+broker.add_middleware(CurrentMessage())
+# allow to save the state of the task
+broker.add_middleware(MessageState(backend=backends.RedisBackend()))
+remoulade.set_broker(broker)
+
+
+@remoulade.actor
+def xor_range(a, b):
```

Comment:
could you make it simpler ? actor count words with mulitple urls maybe ? and just one group. what do you think ?

Reply:
@antoinerabany  okay!